config/main.py

```diff
@@ -1035,6 +1036,10 @@ def add_vlan_member(ctx, vid, interface_name, untagged):
         else:
             ctx.fail("{} is already a member of {}".format(interface_name,
                                                         vlan_name))
+    for entry in interface_table:
+        if (interface_name == entry[0]) and ((entry[1].count('.') == 3) or (entry[1].count(':') >= 2)):
```

Comment:
By defintion, "INTERFACE" table is for L3 entries. So I don't think you need to check for `.` or `:` as it is implicit.

Reply:
you are correct, I have fixed it